maquetado/scss/styles.scss

```diff
@@ -25,62 +25,78 @@ body {
 }
 
 .main-content {
-	display: flex;
-	flex-direction: column;
-	align-items: center;
-	padding: 150px 100px;
+	padding: 50px 100px;
+	max-width: 1300px;
+	margin: 0 auto;
 
 	.title {
-		display: flex;
-		align-self: flex-start;
-		position: relative;
-		left: 0;
-		margin: 30px 40px;
-
-		.section-title {
-			font-size: $font-big;
-			
-			&::before{
-				content: '';
-				height: $triangle-height;
-				width: $triangle-width;
-				background: url('assets/triangle.svg') no-repeat;
-				position: absolute;
-				left: -50px;
-				top: -30px;
-				z-index: -1;
-			}	
+			margin: 120px 0 100px 60px;
+	
+			.section-title {
+				font-size: $font-big;
+				position: relative;
+				top: 30px;
+				left: 60px;
+			}
+			&::before {
+			    background-image: url('assets/triangle.svg') ;
+			    background-size: $triangle-height 90px;
+			    display: inline-block;
+			    width: $triangle-width; 
+			    height: $triangle-height;
+			    content:"";
+			    position: absolute;
+			}
 		}
 	}
 
 	.techs {
-		display: flex;
-		flex-wrap: wrap;
-		justify-content: space-between;
-		padding: 30px 50px;
+		width: 100%;
+		display: grid;
+		grid-template-columns:
```

Comment:
try using the code i commented earlier so there is no need to specify a class to all the columns.